src/components/language/language.js

```diff
@@ -55,9 +57,36 @@ const COMPONENT_NAME = 'Language';
 class Language extends Component {
   state: Object;
   _controlLanguageElement: any;
+  _lastActiveTextLanguage: string = '';
   // ie11 fix (FEC-7312) - don't remove
   _portal: any;
-
+  _keyboardEventHandlers: Array<KeyboardEventHandlers> = [
+    {
+      eventType: 'keydown',
```

Comment:
each event should have 
{
  eventType?,
  key,
  handler
}

cause each key binding can have different eventType potentially
Also, eventType should be optional cause keyDown is the default

Reply:
I grouped by eventType to make it clear
[{eventType:keydown, 
handlers:[]}]

Reply:
Handler -> {key:keyboardKey, action:function}